lib/schema.js

```diff
@@ -324,7 +326,7 @@ function PrimitiveSchema(type) {
 
 util.inherits(PrimitiveSchema, Schema);
 
-function FieldSchema(name, type) {
+function FieldSchema(name, type, defaultValue) {
```

Comment:
The type variable is a Schema object (that now has a default value), but the FieldSchema object also has a default value.  It looks to me like FieldSchema does not need a default value, but the various Schema subclasses do.